sky/clouds/cloud.py

```diff
@@ -239,6 +239,13 @@ def accelerators_to_hourly_cost(self, accelerators: Dict[str, int],
         """Returns the hourly on-demand price for accelerators."""
         raise NotImplementedError
 
+    def instance_type_to_hourly_carbon_cost(self, instance_type: str,
+                                            accelerators: Dict[str, int],
+                                            region: Optional[str],
+                                            zone: Optional[str]) -> float:
+        """Returns the hourly carbon cost for an instance type."""
+        raise NotImplementedError
```

Comment:
Instead of NotImplementedError, should we `return -1` as the default base implementation? I had some other clouds enabled (cudo, paperspace), and had to disable them otherwise I'd get this error:
```
Traceback (most recent call last):
  File "/Users/romilb/tools/anaconda3/bin/sky", line 8, in <module>
    sys.exit(cli())
  File "/Users/romilb/tools/anaconda3/lib/python3.9/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/Users/romilb/tools/anaconda3/lib/python3.9/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/utils/common_utils.py", line 354, in _record
    return f(*args, **kwargs)
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/cli.py", line 828, in invoke
    return super().invoke(ctx)
  File "/Users/romilb/tools/anaconda3/lib/python3.9/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/romilb/tools/anaconda3/lib/python3.9/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/romilb/tools/anaconda3/lib/python3.9/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/utils/common_utils.py", line 375, in _record
    return f(*args, **kwargs)
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/cli.py", line 1143, in launch
    _launch_with_confirm(task,
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/cli.py", line 592, in _launch_with_confirm
    dag = sky.optimize(dag, objective_target=optimize_cost_target_enum)
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/optimizer.py", line 154, in optimize
    unused_best_plan = Optimizer._optimize_dag(
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/optimizer.py", line 1138, in _optimize_dag
    Optimizer._estimate_nodes_cost_or_time(local_topo_order,
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/optimizer.py", line 302, in _estimate_nodes_cost_or_time
    _fill_in_launchable_resources(
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/optimizer.py", line 1395, in _fill_in_launchable_resources
    _make_launchables_for_valid_region_zones(
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/optimizer.py", line 1304, in _make_launchables_for_valid_region_zones
    ci_cost = launchable_res_cp.get_carbon_cost(execution_time)
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/resources.py", line 1007, in get_carbon_cost
    carbs = self.cloud.instance_type_to_hourly_carbon_cost(
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/clouds/cloud.py", line 247, in instance_type_to_hourly_carbon_cost
    raise NotImplementedError
```

Reply:
Done!